eth-erc20-registration/src/main/kotlin/token/ERC20TokenRegistration.kt

```diff
@@ -41,7 +40,7 @@ class ERC20TokenRegistration(
                     Result.of { logger.warn { "No ERC20 tokens to register" } }
                 } else {
                     EthTokensProviderImpl.logger.info { "ERC20 tokens to register $tokensToRegister" }
```

Comment:
It's not your fault, but I can see a little problem here. Logger was taken from another file by mistake.

Reply:
Thank you for detecting this problem